locales/sv-SE.json

```diff
@@ -0,0 +1,336 @@
+    "Clear watch history?": "Töm visningshistorik?",
+    "New password": "Nytt lösenord",
+    "New passwords must match": "Nya lösenord måste stämma överens",
+    "Cannot change password for Google accounts": "Kan inte ändra lösenord på Google-konton",
+    "Authorize token?": "Auktorisera åtkomsttoken?",
+    "Authorize token for `x`?": "Auktorisera åtkomsttoken för `x`?",
+    "Yes": "Ja",
+    "No": "Nej",
+    "Import and Export Data": "Importera och exportera data",
+    "Import": "Importera",
+    "Import Invidious data": "Importera Invidious-data",
+    "Import YouTube subscriptions": "Importera YouTube-prenumerationer",
+    "Import FreeTube subscriptions (.db)": "Importera FreeTube-prenumerationer (.db)",
+    "Import NewPipe subscriptions (.json)": "Importera NewPipe-prenumerationer (.json)",
+    "Import NewPipe data (.zip)": "Importera NewPipe-data (.zip)",
+    "Export": "Exportera",
+    "Export subscriptions as OPML": "Exportera prenumerationer som OPML",
+    "Export subscriptions as OPML (for NewPipe & FreeTube)": "Exportera prenumerationer som OPML (för NewPipe och FreeTube)",
+    "Export data as JSON": "Exportera data som JSON",
+    "Delete account?": "Radera konto?",
+    "History": "Historik",
+    "An alternative front-end to YouTube": "Ett alternativt gränssnitt till YouTube",
+    "JavaScript license information": "JavaScript-licensinformation",
+    "source": "källa",
+    "Log in": "Logga in",
+    "Log in/register": "Logga in/registrera",
+    "Log in with Google": "Logga in med Google",
+    "User ID": "Användar-ID",
+    "Password": "Lösenord",
+    "Time (h:mm:ss):": "Tid (h:mm:ss):",
+    "Text CAPTCHA": "Text-CAPTCHA",
+    "Image CAPTCHA": "Bild-CAPTCHA",
+    "Sign In": "Inloggning",
+    "Register": "Registrera",
+    "E-mail": "E-post",
+    "Google verification code": "Google-bekräftelsekod",
+    "Preferences": "Inställningar",
+    "Player preferences": "Spelarinställningar",
+    "Always loop: ": "Loopa alltid: ",
+    "Autoplay: ": "Autouppspelning: ",
+    "Play next by default: ": "Spela nästa som förval: ",
+    "Autoplay next video: ": "Autouppspela nästa video: ",
+    "Listen by default: ": "Lyssna som förval: ",
+    "Proxy videos: ": "Proxy:a videor: ",
+    "Default speed: ": "Förvald hastighet: ",
+    "Preferred video quality: ": "Föredragen videokvalitet: ",
+    "Player volume: ": "Volym: ",
+    "Default comments: ": "Förvalda kommentarer: ",
+    "youtube": "YouTube",
+    "reddit": "Reddit",
+    "Default captions: ": "Förvalda  undertexter: ",
+    "Fallback captions: ": "Ersättningsundertexter: ",
+    "Show related videos: ": "Visa relaterade videor? ",
+    "Show annotations by default: ": "Visa länkar-i-videon som förval? ",
+    "Visual preferences": "Visuella inställningar",
+    "Player style: ": "Spelarstil: ",
+    "Dark mode: ": "Mörkt läge: ",
+    "Theme: ": "Tema: ",
+    "dark": "Mörk",
+    "light": "Ljus",
+    "Thin mode: ": "Tunt läge: ",
```

Comment:
`Lättviktigt läge` (light-wight mode) would be better, but that's not what it says in English. The mode omits images

Reply:
"Läge utan bilder" or "Visa Inga bilder" or something?